pkg/asset/machines/libvirt/machinesets.go

```diff
@@ -32,8 +31,8 @@ func MachineSets(clusterID string, config *types.InstallConfig, pool *types.Mach
 		total = *pool.Replicas
 	}
 
-	provider := provider(clustername, config.Networking.MachineCIDR.String(), platform, userDataSecret)
-	name := fmt.Sprintf("%s-%s-%d", clustername, pool.Name, 0)
+	provider := provider(clusterID, config.Networking.MachineCIDR.String(), platform, userDataSecret)
+	name := fmt.Sprintf("%s-%s-%d", clusterID, pool.Name, 0)
```

Comment:
What should Hive be passing when we call this function? This looks like it's still cluster ID, but should probably be infraID?

Reply:
N/M found where it was passed in, we're sending infraID, var could probably use a rename though.